sys/userlib/src/lib.rs

```diff
@@ -969,6 +969,13 @@ pub fn sys_irq_control(mask: u32, enable: bool) {
     }
 }
 
+#[inline(always)]
+pub fn sys_irq_control_clear_pending(mask: u32, enable: bool) {
+    unsafe {
+        sys_irq_control_stub(mask, enable as u32 | 0b10);
```

Comment:
it seems like it could be nicer if the bitflags for this argument were defined in `sys/abi` so that the kernel and userlib could depend on the same definition, instead of having magic numbers. this isn't a huge deal, and if it's too annoying, we're probably not going to go and mess with them any time soon, so it's fine-as is. but, it would make me a little happier.